main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"TODOGo/api"
+	"TODOGo/config"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"log"
+	"net/http"
+	"os"
+)
+
+var (
+	portLocal = "7540"
+	vebDir    = "./web"
+)
+
+func main() {
+	config.LoadEnv()
+	config.MakeDB()
+	defer config.CloseDB()
+
+	port := os.Getenv("TODO_PORT")
+	if port == "" {
+		port = portLocal
+	}
+	r := chi.NewRouter()
+
+	r.Use(middleware.Logger)
+	r.Use(middleware.Recoverer)
+
+	// Обработка статических файлов
+	r.Handle("/*", http.FileServer(http.Dir(vebDir)))
+
+	// Обработка маршрута для входа
+	r.Post("/signin", api.SignUpHandler)
```

Comment:
Иначе авторизация не работает
```suggestion
	r.Post("/api/signin", api.SignUpHandler)
```